bins/packages/rfs/rfs.sh

```diff
@@ -1,22 +1,31 @@
-RFS_VERSION="2.0.0"
-RFS_CHECKSUM="d6e1c3036d8b0d7d6df4d67fe17a0502"
-RFS_LINK="https://github.com/threefoldtech/rfs/releases/download/v${RFS_VERSION}/rfs"
+RFS_VERSION_V1="1.1.1"
+RFS_VERSION_V2="2.0.0"
```

Comment:
Since we are at it, can we change this to latest "2.0.0" .Note that changing this will also require changing the RFS_CHECKSUM_V2 hash. You can simply do this by downloading the binary from the release 2.0.1 and do an `md5sum rfs` and then use that value.